test/module/irohad/validation/chain_validation_test.cpp

```diff
@@ -45,17 +27,16 @@ class ChainValidationTest : public ::testing::Test {
     peers = std::vector<std::shared_ptr<shared_model::interface::Peer>>();
   }
 
-  /**
-   * Get block builder to build blocks for tests
-   * @return block builder
-   */
-  auto getBlockBuilder() const {
-    return TestBlockBuilder()
-        .transactions(std::vector<shared_model::proto::Transaction>{})
-        .height(1)
-        .prevHash(hash);
+  auto setupBlock(BlockMock &block) const {
+    EXPECT_CALL(block, height()).WillRepeatedly(Return(1));
+    EXPECT_CALL(block, prevHash()).WillRepeatedly(testing::ReturnRef(hash));
+    EXPECT_CALL(block, signatures()).WillRepeatedly(testing::Return(sigs));
+    EXPECT_CALL(block, payload()).WillRepeatedly(testing::ReturnRef(payload));
   }
 
+  std::array<SignatureMock, 0> sigs;
```

Comment:
Zero sized array?

Reply:
Yep, it supported by the standard
SignatureMock cannot be copied, so other sizes won't work